rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -974,12 +974,12 @@ def test_upgrade_v11_v12(globaldb: GlobalDBHandler, messages_aggregator):
         assert cursor.execute(
             'SELECT COUNT(*) FROM general_cache WHERE key IN (?, ?, ?, ?)',
             (
-                CacheType.VELODROME_POOL_ADDRESS.serialize(),
-                CacheType.VELODROME_GAUGE_ADDRESS.serialize(),
-                CacheType.AERODROME_POOL_ADDRESS.serialize(),
-                CacheType.AERODROME_GAUGE_ADDRESS.serialize(),
+                CacheType.VELODROME_GAUGE_FEE_ADDRESS.serialize(),
+                CacheType.VELODROME_GAUGE_BRIBE_ADDRESS.serialize(),
+                CacheType.AERODROME_GAUGE_BRIBE_ADDRESS.serialize(),
+                CacheType.AERODROME_GAUGE_FEE_ADDRESS.serialize(),
             ),
-        ).fetchone()[0] == 1914
+        ).fetchone()[0] == 0
```

Comment:
why start with 0? It should be as was before, and then at the upgrade, delete + replace the new ones. So use the non-fixed DB at start please. Otherwise there is no point to this part of the test.